tuner/examples/simple/simple_tuner.py

```diff
@@ -75,6 +75,7 @@ def main():
 
     path_config = libtuner.PathConfig()
     path_config.base_dir.mkdir(parents=True, exist_ok=True)
+    path_config._set_tune_output(Path("autotune_output.txt"))
```

Comment:
This appears to be a private function (it starts with an underscore), so we shouldn't call it directly in the current form.